tests/test_ipynb.py

```diff
@@ -139,6 +140,17 @@ def test_cell_magic_with_magic() -> None:
     assert result == expected
 
 
+def test_cell_magic_with_custom_python_magic() -> None:
+    src = "%%custom_python_magic -n1 -n2\nx=2"
+    result = format_cell(
+        src,
+        fast=True,
+        mode=replace(JUPYTER_MODE, python_cell_magics={"custom_python_magic"}),
+    )
+    expected = "%%custom_python_magic -n1 -n2\nx = 2"
+    assert result == expected
```

Comment:
should there also be a test to check that if you pass some custom magic, then the default Python magics still get formatted?

Reply:
Probably, and that when not passing but trying to format custom it isn't formatted 👍

Reply:
No problem, pushed.  :-)